changelog.dd

```diff
@@ -45,6 +45,8 @@ $(BUGSTITLE Library Changes,
         $(XREF conv, to) instead.)
     $(LI $(RELATIVE_LINK2 min-max-element,
     `std.algorithm.searching.{min,max}Element` for ranges have been added.))
+    $(LI $(RELATIVE_LINK2 quantize, Added $(XREF math, quantize), for rounding to
```

Comment:
@aG0aep6G Good catch; that was dumb of me. (It's fixed now.)

Is there somewhere that I can view a preview of the new change log? I poked around a bit in the `CyberShadow/DAutoTest` output, but couldn't find the new log; only the old one from the last official release.